src/lib/api/org/index.ts

```diff
@@ -165,10 +180,17 @@ export async function listProjects(
     );
   }
 
+  const oldSnykApiVariable = process.env.SNYK_API
+  process.env.SNYK_API = process.env.SNYK_API_REST 
+
+  const requestManager = new requestsManager({
```

Comment:
hmm, this means we lose the queueing here. For every org we will now make a new request manager instead of 1. So if we have 7k orgs we will create 7k request managers 🤔 This might affect error handling & rate limiting too as the failed requests are no longer in the same queue.

also deleting like this   `process.env.SNYK_API = process.env.SNYK_API_REST` , what happens if I am running 2 instances of the tool and this one is running this command and resets the env variable and the other one is doing import?

I think we need to instead find a way to change the base of the API call or change request manager to allow it 🤔